src/main/java/quotes/App.java

```diff
@@ -0,0 +1,101 @@
+package quotes;
+
+
+import com.google.gson.Gson;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class App {
+
+
+    public static void main(String[] args) {
+
+        String URL = "https://icanhazdadjoke.com/slack";
+        displayJokesToTerminal(URL);
+    }
+
+
+    public static void displayJokesToTerminal(String URL){
+
+        try {
+            readQuoteFromApi(URL);
+        } catch (IOException e) {
+            readQuoteFromLocalMachine();
+        }
+
+    }
+
+    public static void readQuoteFromApi(String URL) throws IOException {
+
+
+        URL iCanHaveBadJokesUrl= new URL(URL);
+
+        HttpURLConnection connect = (HttpURLConnection) iCanHaveBadJokesUrl.openConnection();
+        connect.setRequestMethod("GET");
+
+        BufferedReader inputString = new BufferedReader(
+                new InputStreamReader(connect.getInputStream()));
+
+        StringBuilder quotesString = new StringBuilder();
+        String firstJokeLine = inputString.readLine();
+
+        while(firstJokeLine != null){
+            quotesString.append(firstJokeLine);
+           firstJokeLine = inputString.readLine();
+        }
+
+        Gson gson = new Gson();
+        DadJokes dadJoke = gson.fromJson(quotesString.toString(), DadJokes.class);
+        System.out.println("dadJoke = " + dadJoke.toString());
+
+
+        // write to json file
+        File localMachineJsonQuotesFile = new File("src/main/resources/quotesDAD.json");
+        FileWriter quoteFileWriter = new FileWriter(localMachineJsonQuotesFile, true);
+
+        BufferedWriter buffWriter = new BufferedWriter(quoteFileWriter);
+        buffWriter.newLine();
+        buffWriter.append(quotesString.toString());
```

Comment:
This part can be simplified;

gson.toJson(dadJoke, quoteFileWriter);
quoteFileWrite.close();